torch/_dynamo/variables/misc.py

```diff
@@ -282,11 +299,93 @@ def reconstruct(self, codegen):
         codegen.foreach(self.args)
         codegen.call_function(len(self.args), False)
 
+        def codegen_attr(name: str) -> None:
+            attr = getattr(self, name)
+            if not istype(attr, ConstantVariable):
```

Comment:
Can you assert that if attr is a ConstantVariable, then it must be None (and not some other ConstantVariable)? Otherwise we'd just drop it